### PR TITLE
Add aspects to curriculum/javascript/core/objects/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/objects/object-part-1.md?ref=d3771f549b1e0a4fe6cfe3edf0478bc0e9766b9f

File: javascript/core/objects/converting-to-and-from-json.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ levels:
 
 type: normal
 
+aspects:
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/objects/converting-to-and-from-json.md]: This is definitely `introduction` level, I would remove the `obscura` and `deep` labels.

File: javascript/core/objects/hasownproperty-method.md [hunk 2]
```diff
@@ -29,7 +29,9 @@ links:
 
 
 aspects:
-  - introduction
```
Comment (ryanwarsaw) [inline javascript/core/objects/hasownproperty-method.md]: I would revert this change, it's definitely an `introduction` insight.

File: javascript/core/objects/keywords-in-hash-keys.md [hunk 3]
```diff
@@ -24,7 +24,9 @@ notes: 'Same issue, not sure if it''s not really really basic.'
 
 aspects:
   - introduction
-
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/objects/keywords-in-hash-keys.md]: I would remove `deep`, it's definitely a misc. detail but it's not a complexity and doesn't require two previous insights to understand.

File: javascript/core/objects/object-part-1.md [hunk 4]
```diff
@@ -31,6 +31,8 @@ links:
 
 aspects:
   - introduction
+  - obscura
```
Comment (ryanwarsaw) [inline javascript/core/objects/object-part-1.md]: This is definitely not `obscura`

File: javascript/core/objects/object-part-2.md [hunk 5]
```diff
@@ -20,7 +20,9 @@ tags:
   - introduction
 
 aspects:
-  - introduction
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/objects/object-part-2.md]: Not `deep` or `obscura`, you're going to need this to do a lot of `introduction` level content we serve.
